ios/Plugin/SecureStoragePluginPlugin.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  SecureStoragePluginPlugin.swift
+//  SecureStoragePluginPlugin
+//
+//  Created by Laszlo Blum on 2023. 10. 27.
+//
+
+import Foundation
+import Capacitor
+
+@objc(SecureStoragePluginPlugin)
+public class SecureStoragePluginPlugin: CAPPlugin {
```

Comment:
We should improve the wording here:
- SecureStoragePlugin is the one used by Capacitor (currently SecureStoragePluginPlugin)
- ScureStoragePluginImpl is the impl (currently SecureStoragePlugin)

Reply:
OK